### PR TITLE
DFAST-201 When disabling make figures with save figures enabled, the figure directory stays editable in the GUI

### PR DESCRIPTION
**Pull request by MRVermeulenDeltares**

### CODE REVIEW

File: dfastmi/gui/dialog_view.py [hunk 1]
```diff
@@ -1069,32 +1077,29 @@ def _update_plotting(self) -> None:
...
+        figure_dir_button.setEnabled(value)
 
     def _update_save_plotting(self) -> None:
         """Update the plotting flags."""
 
-        save_plot_gui = (
-            self._save_plots_edit.isChecked() and self._view_model.model.plotting
-        )
-
-        if self._view_model.model.save_plots != save_plot_gui:
```
Comment (ralph-peelen-deltares) [inline dfastmi/gui/dialog_view.py:1089]: this line should remain i think.

File: dfastmi/gui/dialog_view.py [hunk 2]
```diff
@@ -851,7 +857,9 @@ def _menu_load_configuration(self) -> None:
                     )
                 )
             else:
-                self._update_enabling_plotting_controls()
+                self._update_enabled_of_make_plot_dependent_view_items(
```
Comment (ralph-peelen-deltares) [inline dfastmi/gui/dialog_view.py:860]: this should be removed. The setting of the enabled flags should be done in the view model in load_config

File: dfastmi/gui/dialog_view.py (hunk 1 above)
Comment (MRVermeulenDeltares) [reply dfastmi/gui/dialog_view.py:1089]: reverted back with the discussed change.

File: dfastmi/gui/dialog_view.py (hunk 2 above)
Comment (MRVermeulenDeltares) [reply dfastmi/gui/dialog_view.py]: updated in pair programming

**Comment by MRVermeulenDeltares** [comment]
Tested with running the following cfgs in debug mode:
Qmin_4000_v2.cfg, Qmin_4000_v2_rkm.cfg & Qmin_4000_v2_tide.cfg.

everything seems to function correctly.

**Pull request merged**